.circleci/config.yml

```
@@ -158,6 +158,10 @@ jobs:
           name: Deploy static files to Google Cloud Storage Bucket
           command: |
             gsutil -m cp -r -z "js,css,map,txt" ./static/** "gs://${GOOGLE_CLOUD_BUCKET}/${CIRCLE_TAG/#v}/"
+      - run:
+          name: Deploy introspection json to Google Cloud Storage Bucket
+          command: |
+            gsutil cp "introspection.json" "gs://${GOOGLE_CLOUD_BUCKET}"
```

I think it makes sense to output this file in both the root where you've done it here, and the versioned folder.

```suggestion
            gsutil cp "introspection.json" "gs://${GOOGLE_CLOUD_BUCKET}"
            gsutil cp "introspection.json" "gs://${GOOGLE_CLOUD_BUCKET}/${CIRCLE_TAG/#v}/"
```

This ensures we persist the files for each release by themselves.

---

Also in the `Copy static assets from docker container` step ensure you copy the introspection out of the container.